src/main/pages/setup-kubernetes/kubernetes-multi-user.adoc

```diff
@@ -1,80 +1,87 @@
+* If your cluster is running on MiniKube, type `minikube ip` at the terminal.
+* If your cluster is in the cloud, obtain the hostname or IP address from your cloud provider.
 
-In case you’re specifying a hostname, simply pass it as the value of the `ingressDomain` parameter below.
+In production, you must specify a hostname. For details see https://github.com/eclipse/che/issues/8694[Add an ability to use host-based routing instead of path-based for k8s infras]. If you do not have a hostname (example: during development) and want to use a host-based configuration, use services such as nip.io or xip.io.
 
-If you must use IP address (e.g. your corporate policy prevents you from using nip.io), you would also have to set `isHostBased` to `false`.
+If specifying a hostname, pass it as the value of the `ingressDomain` parameter in the deployment syntax in the following section.
+
+If using the IP address (example: your corporate policy prevents you from using nip.io), you must also set `isHostBased` to `false`.
 
 [id="deploy-syntax"]
-== Deploy Syntax
+== Deployment Syntax
 
-The context of the commands below is `che/deploy/kubernetes/helm/che`
+The context of the commands below is `che/deploy/kubernetes/helm/che`.
 
 [id="to-deploy-with-dedicated-keycloak-as-authentication-server"]
-=== To deploy with dedicated keycloak as authentication server:
+=== Deploying with dedicated Keycloak as authentication server
+
+To deploy multi-user with dedicated Keycloak as authentication server:
 
-* Override default values by changing the values.yaml file and then typing:
+. Override default values by changing the `values.yaml` file. On the command line, run the following command:
 +
 ----
 helm upgrade --install <my-che-installation> --namespace <my-che-namespace> -f ./values/multi-user.yaml ./
 ----
-* Or, you can override default values during installation, using the `--set` flag:
++
+Or, override default values during installation, using the `--set` flag:
 +
 ----
 helm upgrade --install <my-che-installation> --namespace <my-che-namespace> -f ./values/multi-user.yaml --set global.ingressDomain=<my-hostname> --set cheImage=<my-image> ./
 ----
+
 * Master: `https://che-<che-namespace>.domain`
 * Keycloak: `https://keycloak-<che-namespace>.domain`
 * Workspaces servers: `https://server-host.domain`
 
 [id="to-deploy-without-keycloak-in-multi-user-mode"]
-=== To deploy without keycloak in multi-user mode
+=== Deploying without Keycloak in the multi-user mode
 
-In this case you have to supply custom OpenIdConnect provider (see https://github.com/eclipse/che-docs/blob/b2310017b1a75901cbec3b9c665d7ffa1cb23177/src/main/pages/setup-openshift/openshift-config.md[here] for more details), using the following flags:
+Supply the custom OpenIdConnect provider (see https://github.com/eclipse/che-docs/blob/b2310017b1a75901cbec3b9c665d7ffa1cb23177/src/main/pages/setup-openshift/openshift-config.md[here] for more details), using the following flags:
```

Comment:
Should "OpenIdConnect" be styled?

Also, the comma before "using the following flags" is not necessary.

Reply:
That should have been OpenID Connect. I changed that but did not add any styling to it. Also, I changed the location of the parenthetical information.